### PR TITLE
update README and CONTRIBUTING

### PR DESCRIPTION
**Pull request by Amertz08**
Updated contribution guidelines. Added link to them in README

### CODE REVIEW

**Comment by Amertz08** [comment]
Yeah. Let the owner of the PR make changes unless necessary or a Meeting notes PR.

**Comment by cduddy** [comment]
Alright. Should I approve yours? I can't approve mine obviously.

**Comment by Amertz08** [comment]
I was mid making additions and had to revert the document to get your commit then rewrite what I had already wrote. This might of be avoidable on my part but not entirely sure. If you don't think any changes should be made then yes. If you have any suggested additions then no.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ This file sets out to establish common operating procedures when contributing to
 
 ## Issues
 
-All issues should be documented, tagged, and assigned.
+All issues should be documented, tagged, and assigned. They should have clearly stated requirements and be a finite work product.
```
Comment (Amertz08) [inline CONTRIBUTING.md]: You can comment on a line like so

**Comment by Amertz08** [comment]
PING - Needs reviewed

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -18,4 +18,6 @@ $ git checkout -b 1-contribution-guide
 
 ## Pull Requests
 
-Must be reviewed by 1 team member. When branch resolves an issue it should be mentioned in the description `closes #ISSUE` to automatically close the issue on merge. PRs should delete branch on merge.
+Must be reviewed by 2 team members. When branch resolves an issue it should be mentioned in the description `closes #ISSUE` to automatically close the issue on merge. PRs should delete branch on merge.
+
+PRs for meeting notes or contribution guidelines should be reviewed by all team members.
```
Comment (nBukaty) [inline CONTRIBUTING.md]: Does github let us enforce this?

Comment (Amertz08) [reply CONTRIBUTING.md]: No unfortunately you can only require 1 review let alone define requirements for specific files. So we are going to have to self regulate.

**Pull request merged**